### DISCUSSION TITLE
Conditions bug : Block seems to interpret 'or' as an 'and'..

### DISCUSSION THREAD

**Discussion by sparkls0** [AutomaApp/automa#1428] [Q&A]
[https://youtu.be/oK-ttrJ6-EM](url)

Hey there, I'm using a sort of loading system that will retrieve an element text value, and check if it's value equals to 'saved', so we can quit the page. 

I've tried to add multiple languages to it, like french for example

As soon as I add an 'or', it seems to behave as an 'and'

Am I missing something? 

I made a quick video to show you the problem

![CleanShot 2023-10-07 at 14 38 11@2x](https://github.com/AutomaApp/automa/assets/48011728/285dc8e9-8f1d-453e-b7c9-884c4a3124a2)

Works as intended when I only put English : it is looking for 'saved' word. 

![CleanShot 2023-10-07 at 14 38 34@2x](https://github.com/AutomaApp/automa/assets/48011728/0f3d52ca-a5a6-44f0-9f6a-6be69fe4d8ab)

If I add french using an 'or', it won't get out of the condition , even though it does find 'saved' value in the first span. 

![CleanShot 2023-10-07 at 14 39 10@2x](https://github.com/AutomaApp/automa/assets/48011728/5b3b7f57-d754-40ca-babc-5a8616a34b31)

**Comment by Delusionnnn** [comment]
EDIT: Just realized you’re not checking if the value changes from “Saved” but rather into “Saved”. In that case just setup a separate path for when value Does Contain “Saved” or “sauvegardé” and don’t use the fallback at all. You can create as many paths in a command block you want. 

BELOW is for if you’re checking if an element text changes from its original value: 

You can’t setup smth to not contain two things at the same time because it’ll always not contain one of them. For something looking to change value from “Saved” it’ll always not contain the French “sauvegardè” and vice versa and therefore never execute the fallback. 

You would probably instead want to grab the text from the element you’re checking for change (if not doing that already) and assign it a variable like “Saved” and then use the same rule but write {{variables.Saved}} and have just one path with one check so it’ll correctly execute the fallback. That’s the basics at least for a loop for checking when an element text changes values.

**Comment by sparkls0** [comment]
Does make sense! Thanks a lot for your time , I understand now

Indeed, I’ll switch for a different system now

**Reply by Delusionnnn** [reply]
Glad to help, lmk if you need additional help
